onward/app/views/fragments/richLinkBody.scala.html

```diff
@@ -50,9 +50,7 @@ <h1 class="rich-link__title">
                         @fragments.inlineSvg("quote", "icon")
                     }
                 } else {
-                    @if(content.content.cardStyle == Comment ||
-                        (content.content.cardStyle == Feature && content.content.metadata.pillar.nameOrDefault == "sport")
```

Comment:
What happens now to the feature sport cards?

Reply:
I spoke with Ben & Frank and features don't always have the quote, that;s because it is added in the fronts tool. I don't think there's a way of detecting that.

Reply:
Okay as long as we are okay that (I believe) this changes the current functionality, not just garnett.